### PR TITLE
[W5.11][T12-2] Hovan Tan Fang Hao

### PR DESCRIPTION
**Pull request by xShaivan**
Test test

### CODE REVIEW

**Comment by xShaivan** [comment]
Ready for Review.

**Comment by xShaivan** [comment]
Edited. Ready for review again.

**Comment by brandonccm1996** [comment]
You can also update the test cases and user guide to reflect that the name and email have been changed to uppercase for better user clarity.

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -26,7 +26,8 @@ public Name(String name) throws IllegalValueException {
         if (!isValidName(trimmedName)) {
             throw new IllegalValueException(MESSAGE_NAME_CONSTRAINTS);
         }
-        this.fullName = trimmedName;
+
+        this.fullName = trimmedName.toUpperCase();
```
Comment (marlenekoh) [inline src/seedu/addressbook/data/person/Name.java:30]: Try not to manipulate data that user has typed. Instead, when implementing case insensitive find, try to use Java's `String .toLowerCase()` method or Java's `Collections.sort()`  with `String.CASE_INSENSITIVE_ORDER`, which will not directly modify stored data (:

**Review by marlenekoh — COMMENTED** [review]
@xShaivan Good job using branching and naming commits (:

Please close this PR after reading review comments

**Pull request closed without merging**